src/main/java/com/baidu/hugegraph/base/ToolClient.java

```diff
@@ -40,7 +40,7 @@ public ToolClient(ConnectionInfo info) {
         this.client = HugeClient.builder(info.url, info.graph)
                                 .configUser(info.username, info.password)
                                 .configTimeout(info.timeout)
-                                .configSSL(info.protocol, info.trustStoreFile,
+                                .configSSL(info.trustStoreFile,
```

Comment:
set trustStoreFile to default value if info.url starts with 'https' and trustStoreFile is empty